### DISCUSSION TITLE
Norwegian translation for the WordPress plugin

### DISCUSSION THREAD

**Discussion by soderlind** [plausible/analytics#3197] [Anything Else]
I just added a Norwegian translation for the WordPress plugin, please review and add it :)

Its at https://translate.wordpress.org/projects/wp-plugins/plausible-analytics/stable/nb/default/

**Comment by soderlind** [comment]
@metmarkosaric ^^ please, my users prefer Norwegian.

**Reply by metmarkosaric** [reply]
@soderlind thank you! do I have the option to add it at all? don't see any obvious way to do so when I check the translate page

**Reply by soderlind** [reply]
@metmarkosaric Someone has, most likely the one that added the plugin.

**Reply by soderlind** [reply]
@metmarkosaric Fixed it myself, asked and became General Translation Editor for `nb_NO` (Norwegian bokmål)

**Reply by metmarkosaric** [reply]
ahh there you go. I'm the account that added the plugin but couldn't see the option to approve a translation so glad you found a way. thank you again for your help with this!

**Reply by soderlind** [reply]
@metmarkosaric Very happy with Plausible, I'm tech lead for a government WordPress mulitisite, running 115 sites with Plausible :) Will, when I'm back from vacation late August, release our "one click add Plausible" add-on for WordPress multisite.

**Reply by metmarkosaric** [reply]
@soderlind really happy to hear that, thanks for sharing! we have some interesting plans for the next plugin release which would include a way to set everything up directly in the plugin settings (shared links, goals etc) without needing to go to plausible.io at all. stay tuned and enjoy Plausible!

**Reply by soderlind** [reply]
@metmarkosaric Glad to hear. Please make the settings available through hooks.

**Reply by metmarkosaric** [reply]
@soderlind I've added a note about hooks so we will consider it

**Comment by Dan0sz** [comment]
@soderlind As of 2.0.5 the `plausible_analytics_settings` filter is available, which allows you to persist settings across all subsites by hooking into it.

In 2.1.0 I'm planning to add a WP CLI command, which allows you to programmatically create and add API tokens. This should make the onboarding process for (large) multisites much easier.
